design/ui/pagination/README.mdx

```diff
@@ -0,0 +1,170 @@
+---
+name: Pagination
+---
+
+import semanticUiCssOffline from 'semantic-ui-css-offline';
+import { Playground, Props } from 'docz';
```

Comment:
We will probably be building our own Props component that better defines what the props are. I think it will also exist in a drawer on the right side of the docs so you can see the props at all times while using a Playground. The Docz Props component also does not tell us what the enum values are, which feels like a big missing piece that we could add with our own. 

We will also probably build our own Playground component with react-live. My hope is that the playground will auto generate all of the playgrounds from a Cosmos fixture named export. So if we have 6 fixtures exported from a single file, our Playground component would create all of them in our doc too. 

Both of the above custom components for the docs would in theory auto import the component and fixture based on the component name defined in the head. Or you could manually define the component too if you needed to.

The reasoning behind this investment is I think we hope that also UX designers will help with this documentation. If we can make it simpler to create this content, and keep it consistent, that is going to pay off for us.